src/odb/src/zutil/util.cpp

```diff
@@ -49,15 +49,19 @@ using std::vector;
 RUDYCalculator::RUDYCalculator(dbBlock* block) : block_(block)
 {
   gridBlock_ = block_->getDieArea();
+  if (gridBlock_.area() == 0) {
```

Comment:
Shouldn't this be an error? I can't think of a case where the die area would be zero.

Reply:
I ran into trouble with opening the GUI in the floorplan step as the rudy object is created before the flooplan has actually been run.